SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -5,6 +5,7 @@
 #include "CoreMinimal.h"
```

Comment:
As in `GridBasedLBStrategy.h`, I believe it's better to include UnrealGDK headers first and then UE4 headers. That would make the dependencies on UE4 explicit.